storagemodule.py

```diff
@@ -0,0 +1,57 @@
+#Question: Do you want me to make this whole thing a module or lots of submodules ? 
+
+#Import libaries
+import os
+import shutil
+import earthpy as et
+
+#Make a directory in any os system's home directory 
+et.io.HOME
+#os.path.exists(et.io.HOME) #a check to see if making it was successful 
+aronspath=os.path.join(et.io.HOME, "arons-kit")#Make the main directory called arons-kit in home directory 
+
+os.mkdir(aronspath)# need to do some conditional statements here if the directory already exists.
+os.chdir(aronspath)#change path to this directory 
```

Comment:
these 3 line should be in a function called something like createDownloadsFolder. Using Aarons should be avoided as this is the name for now, if the business ever changed its name new developers wouldn't know who aaron is :)